lib/binance/config.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Binance.Config do
+  require Logger
+
+  @type t :: %Binance.Config{
+          api_key: String.t(),
+          api_secret: String.t()
+        }
+
+  @enforce_keys [:api_key, :api_secret]
+  defstruct [:api_key, :api_secret]
+
+  @doc """
+  Get default API configs
+
+  ## Examples
+      iex> Binance.Config.get()
+  """
+  def get(nil) do
```

Comment:
Calling `get(nil)` seems like a weird API to me.

I think we should have this as an optional argument to be able to call `get/0`